data-processing/common/commands/fetch_s2_data.py

```diff
@@ -99,19 +101,29 @@ def process(self, item: ArxivId) -> Iterator[S2Metadata]:
                 data = resp.json()
                 references = []
                 for reference_data in data["references"]:
-                    authors = []
-                    for author_data in reference_data["authors"]:
-                        authors.append(Author(author_data["authorId"], author_data["name"]))
-                    reference = Reference(
-                        s2_id=reference_data["paperId"],
-                        arxivId=reference_data["arxivId"],
-                        doi=reference_data["doi"],
-                        title=reference_data["title"],
-                        authors=authors,
-                        venue=reference_data["venue"],
-                        year=reference_data["year"],
-                    )
-                    references.append(reference)
+
+                    if reference_data["paperId"]:
```

Comment:
I don't see this check in the existing code. Is there a reason we're doing it now?

Reply:
Kyle asked the same question, here's the [response](https://github.com/allenai/scholarphi/pull/381#discussion_r1151235345)